modules/rn_3_10_0.adoc

```diff
@@ -8,7 +8,7 @@ Issued 2023-XX-XX
 [id="release-cadence-310"]
 == {productname} release cadence 
 
-With the release of {productname} 3.10, the product will begin to align its release cadence and lifecycle with {ocp}. As a result, {productname} releases will be generally available (GA) within approximately four weeks of the most recent version of {ocp}. Customers can not expect the support lifecycle phases of {productname} to align with {ocp} releases. 
+With the release of {productname} 3.10, the product has begunto align its release cadence and lifecycle with {ocp}. As a result, {productname} releases are now generally available (GA) within approximately four weeks of the most recent version of {ocp}. Customers can not expect the support lifecycle phases of {productname} to align with {ocp} releases. 
```

Comment:
nit: space between begun and to.

Reply:
Thanks! I addressed this in a separate PR.